sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryContentClientTests.java

```diff
@@ -369,7 +370,7 @@ public void uploadFails() {
         BiFunction<HttpRequest, Integer, HttpResponse> onChunk = (r, c) -> {
             if (c == 3) {
                 HttpHeaders responseHeaders = new HttpHeaders().add("Content-Type", String.valueOf("application/json"));
-                String error = "{\"errors\":[{\"code\":\"BLOB_UPLOAD_INVALID\",\"message\":\"blob upload invalid\"}]}";
+                String error = "{\"errors\":[{\"code\":\"BLOB_UPLOAD_INVALID\",\"message\":\"blob upload invalid\"}, {\"code\":\"BLOB_UPLOAD_FOO\",\"message\":\"blob upload foo\"}]}";
```

Comment:
I don't see where you are testing the that caller can at least view the second error code/message - but maybe I am missing it?

Reply:
I'm not testing it - we're logging error responses in core for all failures.